### ISSUE TITLE
Phone number region

### ISSUE THREAD

**Issue by DarkProtocol** [benhurott/react-native-masked-text#23]
Is it possible set phone region in tour module?
For example: +7 (999) 999 99-99
'+7' is region
```javascript
<TextInputMask 
	style={styles.input}
	type={'custom'}
	options={{
		mask: '+7 (999) 999-99-99',
	}} 
	placeholder="Телефон"
	placeholderTextColor="#DBCCC0"
	selectionColor="#CF0821"
	underlineColorAndroid="transparent"
	keyboardType="numeric"
	value={this.state.phoneValue}
	onChangeText={(text) => {
		this.phone = text;	
		this.setState({phoneValue: text});					
		this.setState({phone: false});					
	}}
/>
```
But, when i change input, i see this
![image](https://cloud.githubusercontent.com/assets/22779777/26279634/d18146ec-3dc1-11e7-8c6e-d556fad6869b.png)

**Comment by benhurott** [comment]
Hi,

I will fix this in the next release =).
Thanks for reporting.

**Comment by benhurott** [comment]
Fixed in 1.5.0 =)
